src/types.ts

```diff
@@ -34,18 +32,31 @@ export interface Config {
   provider: ethers.providers.Provider;
 }
 
+export type Maybe<T> = T | undefined;
+
 export interface Listing {
   price: number;
   holder: string;
 }
 
-export interface ZAuctionInstances {
-  [registrarAddress: string]: zAuction.Instance;
+export interface Map<T> {
+  [key: string]: T;
 }
 
-export interface TokenPriceInfo {
-  price: number;
+export interface TokenInfo {
```

Comment:
This interface appears to be identical to the `TokenDto` type declared above in this PR.

Reply:
Modified to remove the duplication here